Sources/RoktUXHelper/Services/LayoutTransformer/LayoutTransformer.swift

```diff
@@ -298,7 +298,7 @@ struct LayoutTransformer<Expander: PayloadExpander, Extractor: DataExtractor> wh
     }
 
     func getDataImage(_ imageModel: DataImageModel<WhenPredicate>, slot: SlotOfferModel?) throws -> DataImageViewModel {
-        let creativeImage = slot?.offer?.creative.images?[imageModel.imageKey]
+        let creativeImage = slot?.offer?.creative.images?[imageModel.imageKey] ?? slot?.offer?.catalogItems?.first?.images?[imageModel.imageKey] // should have only 1 catalogItem at this node
```

Comment:
Nit: No need to address it in this PR as you will work on the next part in a separate PR. I think we need to look into both creative image anc CatalogItem images based on the context as the name could exists on both images.